runtime/autoload/health/provider.vim

```diff
@@ -682,6 +738,9 @@ function! health#provider#check() abort
   call s:check_clipboard()
   call s:check_python(2)
   call s:check_python(3)
+  if exists('$VIRTUAL_ENV')
+    call s:check_active_virtualenv()
+  endif
```

Comment:
let's not start adding control flow logic here, keep it in the `s:check_xx` functions.